### DISCUSSION TITLE
Need Help! PB inactive/afk

### DISCUSSION THREAD

**Discussion by Legan79** [mod-playerbots/mod-playerbots#640] [Q&A] [answered]
Hello everyone,
I urgently need help with my problem, which I've been having for some time. All player bots are now completely inactive/afk and no longer level up.
When I bring them into the group everything goes normally.
I've already tried everything possible in the config file.
Does anyone have any ideas?

![Screenshot 2024-10-24 200938](https://github.com/user-attachments/assets/403e4ae5-42cb-4957-aa2b-4c221ec007ca)
[Playerbot Config.txt](https://github.com/user-attachments/files/17511417/Playerbot.Config.txt)

**Comment by Celandriel** [comment]
What that commit version of your core and mod?

**Comment by noisiver** [comment, marked as answer]
Try setting `botActiveAloneSmartScale` to 0 and `BotActiveAlone` to 100.

**Reply by Legan79** [reply]
The botActiveAloneSmartScale-value was the problem.
Now it works fine again.
Thank you very much!!
